src/tsconfig.json

```diff
@@ -24,6 +25,13 @@
       "es2018",
       "dom"
     ],
-    "preserveSymlinks": true
+    "preserveSymlinks": true,
+    "paths": {
+      "@acme/extensions": ["frontend/packages/acme_extensions/src/public-api.ts"],
```

Comment:
Not sure this path would be correct when included?

Reply:
Removed